### ISSUE TITLE
Another project on Southey's Amadis

### ISSUE THREAD

**Issue by ebeshero** [ebeshero/Amadis-in-Translation#11]
I discovered right here on GitHub another project on Southey's 1803 translation of Amadis. It's just a simple effort to produce a good machine-readable copy of the text, but I don't know how far the researcher has gotten, and I want to inspect it more closely. Here's the repository: https://github.com/swalepoole/amadis-of-gaul

I'm marking an Issue here so I remind myself to go back and take a closer look! If the text is good, I won't have to be keying in the Southey. If it's not good, I might just offer the other archive my files.

**Comment by setriplette** [comment]
That could potentially save us lots of time!

Do you have a link?

Sent from my iPhone

On Aug 29, 2015, at 9:40 PM, Elisa Beshero-Bondar <notifications@github.com<mailto:notifications@github.com>> wrote:

I discovered right here on GitHub another project on Southey's 1803 translation of Amadis. It's just a simple effort to produce a good machine-readable copy of the text, but I don't know how far the researcher has gotten, and I want to inspect it more closely. Here's the repository: https://github.com/swalepoole/amadis-of-gaul

##

**Comment by ebeshero** [comment]
It's all in their repository: https://github.com/swalepoole/amadis-of-gaul
I haven't had a chance to see if those are just OCR'd texts or if they are cleaned up...It looks like they stashed the files there sometime ago and I'm not sure how much processing has been done.

E

**Comment by ebeshero** [comment]
The Swalepool GitHub edition of Southey's Amadis is all written in markdown(!) The main text _appears_ mostly the same as the 1803 text we are using, but it includes embellished abstracts and chapter titles which are lacking in the edition we are using. I suspect those embellished abstracts and titles come from later editions, but Swalepool doesn't precisely indicate which edition is being represented. And there are occasional errors from OCR. I think this is an attempt to clean up a dirty OCR text (but it's not clear which one).

Still, it's close enough to save us time. I used it as my base for coding Ch. 2, and had to make one or two corrections as I compared it to the Southey 1803 photofacsimile we're working with. Careful proofreading is necessary (but would be anyway, even in typing in the text.  So, we'll Use it with Caution.
